include/dmlc/build_config.h

```diff
@@ -0,0 +1,8 @@
+/* default detection logic for fopen64; may be overriden by CMake */
```

Comment:
add include guard as in other DMLC header files

Reply:
Done